cypress/e2e/tests/migration/applicationinventory/assessment/questionnaire_features.test.ts

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    deleteByList,
+    clickItemInKebabMenu,
+    clickByText,
+    createMultipleStakeholders,
+    click,
+    clickJs,
+} from "../../../../../utils/utils";
+import { Stakeholders } from "../../../../models/migration/controls/stakeholders";
+import { AssessmentQuestionnaire } from "../../../../models/administration/assessment_questionnaire/assessment_questionnaire";
+import { confirmButton, nextButton } from "../../../../views/common.view";
+import {
+    legacyPathfinder,
+    cloudNative,
+    SEC,
+    button,
+    cloudReadinessQuestionnaire,
+    cloudReadinessFilePath,
+} from "../../../../types/constants";
+import { Application } from "../../../../models/migration/applicationinventory/application";
+import { Assessment } from "../../../../models/migration/applicationinventory/assessment";
+import * as data from "../../../../../utils/data_utils";
+import { questionBlock } from "../../../../views/assessment.view";
+
+let stakeholderList: Array<Stakeholders> = [];
+
+describe(["@tier3"], "Tests related to questionnaire features", () => {
+    before("Import and enable Cloud readiness questionnaire template", function () {
+        login();
+
+        AssessmentQuestionnaire.deleteAllQuestionnaires();
+        AssessmentQuestionnaire.disable(legacyPathfinder);
+        AssessmentQuestionnaire.import(cloudReadinessFilePath);
+        AssessmentQuestionnaire.enable(cloudReadinessQuestionnaire);
+        stakeholderList = createMultipleStakeholders(1);
+    });
+
+    it("1) Test conditional questions based on existing tags for applications; 2) Cancel assessment", function () {
+        //Automates Polarion MTA-385 Test conditional questions based on existing tags for applications
```

Comment:
This is not required again : 
`Test conditional questions based on existing tags for applications`